regex-syntax/src/hir/translate.rs

```diff
@@ -1040,7 +1040,7 @@ fn ascii_class(kind: &ast::ClassAsciiKind) -> &'static [(char, char)] {
             X
         }
         Blank => {
-            const X: T = &[(' ', '\t')];
+            const X: T = &[('\t', '\t'), (' ', ' ')];
```

Comment:
Oof. Thanks for catching this! It looks like a bad transcription error. However, I don't think this is right either. The docs include more characters:

```
[[:space:]]    whitespace ([\t\n\v\f\r ])
```

And this is consistent with how `regex-syntax 0.4` and below behaved:

https://github.com/rust-lang/regex/blob/fa5cf6b86e38d010de816ad4e2a096a8c62e7640/regex-syntax/src/parser.rs#L1453-L1454

Reply:
@BurntSushi You're referring to docs for `space`, but this bug and fix are for `blank` which has fewer characters.

Reply:
Ug. Thanks for that. I don't think I had quite woken up yet!